TombEngine/Objects/Generic/Switches/airlock_switch.cpp

```diff
@@ -0,0 +1,82 @@
+#include "framework.h"
```

Comment:
Please alphabetise includes. :)
Also, since this is a new file, it can use Pascal case: `AirlockSwitch.cpp/h`.

Reply:
done